src/components/Task/Modal/Form/Complete/TaskModalCompleteForm.test.tsx

```diff
@@ -359,49 +339,44 @@ describe('TaskModalCompleteForm', () => {
 
   it('saves complex', async () => {
     const completedAt = DateTime.local(2015, 1, 5, 1, 2).toISO();
-    const taskAttributes = {
-      id: task.id,
+
+    const taskOverrides = {
+      activityType: ActivityTypeEnum.FollowUpPhoneCall,
       completedAt,
-      displayResult: DisplayResultEnum.FollowUpResultPartnerSpecial,
-      result: ResultEnum.Completed,
-      nextAction: ActivityTypeEnum.PartnerCareThank,
       tagList: ['tag-1', 'tag-2'],
     };
-    const { findByRole, getByText } = render(
-      <Components
-        mocks={[
-          completeTaskMutationMock(accountListId, taskId),
-          GetThisWeekDefaultMocks()[0],
-          {
-            request: {
-              query: CompleteTaskDocument,
-              variables: {
-                accountListId,
-                attributes: taskAttributes,
-              },
-            },
-            result: {
-              data: {
-                updateTask: {
-                  task: taskAttributes,
-                },
-              },
-            },
-          },
-        ]}
-        taskOverrides={{
-          activityType: ActivityTypeEnum.FollowUpPhoneCall,
-          completedAt,
-          tagList: ['tag-1', 'tag-2'],
-        }}
-      />,
+    const { getByText, findByText, getByRole } = render(
+      <Components taskOverrides={taskOverrides} />,
     );
-    userEvent.click(await findByRole('combobox', { name: 'Result' }));
-    userEvent.click(await findByRole('option', { name: 'Partner-Special' }));
-    userEvent.click(await findByRole('combobox', { name: 'Next Action' }));
-    userEvent.click(await findByRole('option', { name: 'Thank You Note' }));
+
+    await waitFor(() => {
```

Comment:
Let's not do a manual 1 second wait. Is the next `waitFor` timing out? If so, you can pass a longer timeout to it `waitFor(() => ..., { timeout: 5000 })` https://testing-library.com/docs/dom-testing-library/api-async/#waitfor